recipes/lua/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(lua REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} lua::lua)
+if(COMPILE_AS_CPP)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE COMPILE_AS_CPP)
+endif()
```

```suggestion
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```
So we can re-use from test_package/Cmakelists.txt